index.html

```diff
@@ -2,7 +2,7 @@
 <head>
     <title>Leaflet Draw Documentation</title>
     <script>
-        location = "http://leaflet.github.io/Leaflet.draw/docs/leaflet-draw-latest.html";
```

Comment:
Changing my master branch will break my github.io page that demonstrates the fix working.

I think it's probably best to re-create this PR from a new branch in my repo that doesn't have any change to the index.html then it can be easily merged into the master branch here,